src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx

```diff
@@ -47,36 +53,91 @@ class FiltersAndSampling extends AsyncView<Props, State> {
   }
 
   componentDidMount() {
-    this.getTransactionsRules();
+    this.getRules();
   }
 
-  componentDidUpdate(prevProps: Props, prevState: State) {
-    if (
-      !isEqual(
-        this.props.organization.dynamicSampling,
-        prevProps.organization.dynamicSampling
-      )
-    ) {
-      this.getTransactionsRules();
-    }
-    super.componentDidUpdate(prevProps, prevState);
-  }
-
-  getTransactionsRules() {
-    const {organization} = this.props;
-    const {dynamicSampling} = organization;
-
-    const transactionRules = dynamicSampling.filter(
-      rule => rule.ty !== DynamicSamplingRuleType.ERROR
+  getRules() {
+    const dynamicSampling: DynamicSamplingRules = [
+      {
+        condition: {
+          operator: 'globMatch',
+          name: 'releases',
+          value: ['1.1.1', '1.1.2'],
+        },
+        sampleRate: 0.7,
+        ty: 'trace',
+      },
+      {
+        condition: {
+          operator: 'and',
+          inner: [
+            {
+              operator: 'strEqualNoCase',
+              name: 'environments',
+              value: ['dev', 'prod'],
+            },
+            {
+              operator: 'strEqualNoCase',
+              name: 'userSegements',
```

Comment:
```suggestion
              name: 'userSegments',
```

Reply:
I will delete it soon...it's just a mock :)